extension/DiscordEmbedBuilder/DiscordEmbedBuilder.csproj

```diff
@@ -248,22 +94,34 @@
   <ItemGroup>
     <None Include="app.config" />
     <Compile Include="Discord.Types.cs" />
-    <None Include="packages.config" />
   </ItemGroup>
   <ItemGroup>
     <Content Include="FodyWeavers.xml" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Costura.Fody">
+      <Version>5.7.0</Version>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
```

Comment:
https://learn.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#controlling-dependency-assets

Reply:
Creates this warning on build. Doesn't seem to cause an issue but I'll put it here for the record:
>MSBUILD : warning FodyPackageReference: Fody: The package reference for Costura.Fody is missing the 'compile' part in the IncludeAssets setting; it's recommended to completely remove IncludeAssets